.github/workflows/ci.yaml

```diff
@@ -167,6 +167,7 @@ jobs:
           - amass
           - angularjs-csti-scanner
           - cmseek
+          - ffuf
```

Comment:
Also include ffuf in `.github/workflows/release-build.yaml` as so:
This is always required because we always build our own parsers.
```yaml
  parsers:
    name: "Build | Parsers"
    needs: sdk
    runs-on: ubuntu-latest
    strategy:
      matrix:
        parser:
          - ...
          - ffuf
          	
```
This is only required for scanners that do not have their own Docker image (as is the case with ffuf)
```yaml
  scanners-third-party:
    name: "Build | Third Party Scanner"
    runs-on: ubuntu-latest
    strategy:
      matrix:
        scanner:
          - ...
          - ffuf
```
New repos also have to created for securecodebox/scanner-ffuf and  securecodebox/parser-ffuf  in Dockerhub so they can be uploaded in the release build. @J12934 is one the people who has the permissions to do that, I don't.

Reply:
Will add ffuf to release-build.yaml. @J12934 can you do the Dockerhub part?

Reply:
@rseedorff can you add @J12934 to the Dockerhub group?